spec/pageflow/revision_component_spec.rb

```diff
@@ -12,23 +12,6 @@ module Pageflow
       }.to(change { revision.reload.updated_at })
     end
 
-    describe '#perma_id' do
-      it 'is set on creation' do
-        revision = create(:revision)
-        revision_component = TestRevisionComponent.create!(revision: revision)
-
-        expect(revision_component.perma_id).to be_present
-      end
-
-      it 'differs for separate RevisonComponents' do
-        revision = create(:revision)
-        revision_component1 = TestRevisionComponent.create!(revision: revision)
-        revision_component2 = TestRevisionComponent.create!(revision: revision)
-
-        expect(revision_component1.perma_id).not_to eq(revision_component2.perma_id)
-      end
-    end
-
```

Comment:
I'd vouch for letting these specs stay since `RevisionComponent` is part of the public API and the perma id functionality is sort of part of it's contract. A little duplication in specs never hurts.